src/lib/update-alternatives/UI/main_dialog.rb

```diff
@@ -37,25 +42,38 @@ def dialog_content
     end
 
     def show_handler
-      AlternativesDialog.new.run
+      open_alternatives_dialog
```

Comment:
NP: you can define simple alias `alias show_handler open_alternatives_dialog` instead of the method definition if you do not plan to add anything more to the method.